.github/workflows/ci.yml

```diff
@@ -231,12 +231,12 @@ jobs:
         run: |
           ALL_FEATURES=$(cargo metadata --format-version=1 --no-deps | jq -r '.packages[] | select(.name == "libp2p") | .features | keys | map(select(. != "full")) | sort | join(" ")')
           FULL_FEATURE=$(cargo metadata --format-version=1 --no-deps | jq -r '.packages[] | select(.name == "libp2p") | .features["full"] | sort | join(" ")')
-          
+
           test "$ALL_FEATURES = $FULL_FEATURE"
-          
+
           echo "$ALL_FEATURES";
           echo "$FULL_FEATURE";
-          
+
```

Comment:
Sorry, these are just my editor stripping trailing whitespace. If they bother you, I can look into making it not do this for this file.

Reply:
That is fine, thanks! :)